src/Main.css

```diff
@@ -11,7 +11,7 @@ body {
  * Make most of the text elements use the defined text color.
  * This list may be incomplete
  */
-p, b, h1, h2, h3, h4, h5, em, span:not(.fa), div:not(class^="ant-tooltip"), a:not(.link), li {
+p, b, h1, h2, h3, h4, h5, em, span:not(.fa), span, div:not(.ant-tooltip-inner), a:not(.link), li {
```

Comment:
span selector is dublicated (`span:not(.fa)` already includes all spans excepting that with `.fa` class set)

Reply:
Good catch! Fixed via https://github.com/terrestris/react-geo-baseclient/commit/3e10e743950074b039d211114506b4150025e647